### DISCUSSION TITLE
Installing Labgrid as dependency

### DISCUSSION THREAD

**Discussion by snerzkristall** [labgrid-project/labgrid#1002] [Q&A] [answered]
I am trying to build a python package with labgrid as dependency. The package has some custom drivers/protocols/tests. I can't create a PR because it's company property, but I would like to be able update to newer versions of labgrid, so forking is not an option.

Client-side install (labgrid-venv) is not a problem, but server-side I need crossbar-requirements.txt as well (in crossbar-venv).
Default pip installation just uses requirements.txt. Is there a good way to do this?

**Comment by Bastian-Krause** [comment, marked as answer]
When #1001 is merged, we will recommend installation via pip. The *-requirements.txt will become extras installable via `pip install labgrid[crossbar,..]`. Will this solve your problem?

If not, please tell us what exactly you would need to get your use case working.

**Reply by snerzkristall** [reply]
Yes that would be perfect!
